src/Tizen.NUI.Components/Controls/Button.cs

```diff
@@ -163,8 +171,15 @@ public Button(ButtonStyle buttonStyle) : base(buttonStyle)
         /// An event for the button clicked signal which can be used to subscribe or unsubscribe the event handler provided by the user.<br />
         /// </summary>
         /// <since_tizen> 6 </since_tizen>
+        [Obsolete("Deprecated in API8; Will be removed in API10. Please use Clicked")]
         public event EventHandler<ClickEventArgs> ClickEvent;
 
+        /// <summary>
+        /// An event for the button clicked signal which can be used to subscribe or unsubscribe the event handler provided by the user.<br />
```

Comment:
The line break `<br/>` at the end of text is unnecessary. If it was a multi line text then it would make a difference, but currently `<summary>` tag is just enough. Also I am not sure whether it would be ignored during the rendering phase by the IDE or it would look inconsistent with the rest of the documentation.
```suggestion
        /// An event for the button clicked signal which can be used to subscribe or unsubscribe the event handler provided by the user.
```

Reply:
Thank you for your comment. I have updated the code based on your comment.